tests/internal/test_settings.py

```diff
@@ -182,7 +196,7 @@ def test_settings_missing_lib_config(config, monkeypatch):
     base_rc_config = _base_rc_config({})
 
     # Delete "lib_config" from the remote config
-    del base_rc_config["config"][0]["lib_config"]
+    del base_rc_config["config"][1]["lib_config"]
```

Comment:
Do you know why we originally tested for this? 🤔

Reply:
We're not really "testing for this" per say, I guess we just wanted to delete the lib_config, and then see what it does? In this case we're doing the same thing, it's just that lib_config lives one list item later.